deps/deps.go

```diff
@@ -462,6 +467,45 @@ func GetConfig(ctx context.Context, layers []string, db *harmonydb.DB) (*config.
 	return curioConfig, nil
 }
 
+func updateBaseLayer(ctx context.Context, db *harmonydb.DB) error {
```

Comment:
Should do the upgrade with one db transaction

Reply:
Fixed